src/config.rs

```diff
@@ -17,6 +20,8 @@ static CONFIG_FILE: Lazy<String> = Lazy::new(|| {
     get_env("CONFIG_FILE").unwrap_or_else(|| format!("{data_folder}/config.json"))
 });
 
+pub static SKIP_CONFIG_VALIDATION: Lazy<AtomicBool> = Lazy::new(|| AtomicBool::new(false));
```

Comment:
I'm pretty sure we can skip the `Lazy` here, `AtomicBool::new` should be able to be const initialized, right?
```suggestion
pub static SKIP_CONFIG_VALIDATION: AtomicBool = AtomicBool::new(false);
```

Reply:
Ill check again, but i thought i had tried that.

Reply:
Strange, i really tried this, not sure why it didn't worked, or something complained maybe.
I found it strange my self too at that time.